xml/operations-configuring-identity-websso.xml

```diff
@@ -322,22 +322,25 @@
    </listitem>
    <listitem>
     <para>
-     Copy adfs_metadata.xml to the &clm; node in your preferred
-     location. Here it is /tmp.
+     Copy adfs_metadata.xml to the &clm; node and place it into <filename>/var/lib/ardana/openstack/my_cloud/config/keystone/</filename> directory and put it under revision control.
```

Comment:
Could you please wrap this text to make it easier to review? :)

Reply:
Done